docs/en/Running-Hive-with-Alluxio.md

```diff
@@ -182,13 +182,28 @@ Add the following property to `hive-site.xml` in your Hive installation `conf` d
 </property>
 ```
 
-To use fault tolerant mode, set Alluxio scheme to be `alluxio-ft`:
+To use fault tolerant mode, set the Alluxio cluster properties appropriately in an
+`alluxio-site.properties` file which is on the classpath.
+
+```properties
```

Comment:
Created https://alluxio.atlassian.net/browse/ALLUXIO-2985 since we use this throughout our docs.